VideoLocker/res/layout/row_myvideo_course_list.xml

```diff
@@ -4,41 +4,39 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
 
-    <LinearLayout
+    <RelativeLayout
         android:id="@+id/course_row_layout"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="@dimen/course_card_height"
         android:background="@color/white"
-        android:descendantFocusability="blocksDescendants"
-        android:orientation="vertical"
-        android:paddingBottom="11dp" >
+        android:descendantFocusability="blocksDescendants" >
 
-        <com.android.volley.toolbox.NetworkImageView
+        <ImageView
             android:id="@+id/course_image"
             android:layout_width="match_parent"
-            android:layout_height="120dp"
-            android:adjustViewBounds="true"
-            android:contentDescription="@string/course_image"
-            android:scaleType="centerCrop" />
+            android:layout_height="match_parent"
+            tools:background="@drawable/sample_course_image" />
 
         <RelativeLayout
```

Comment:
This `RelativeLayout` could probably be converted into a `LinearLayout`. Alternatively, with a few changes in the logic, it could be removed, and it's children added directly into the base `RelativeLayout`.